packages/pilot/src/containers/TransactionsList/tableColumns.js

```diff
@@ -50,6 +53,18 @@ const getDefaultDocumentNumber = pipe(
   )
 )
 
+const applyTruncateCustomerEmail = (item) => {
+  const value = path(['customer', 'email'], item)
+
+  return value
+    ? (
+      <Truncate
+        text={value}
+      />
+    )
+    : ''
```

Comment:
acho que aqui faz sentido retornar null
```suggestion
    : null
```

Reply:
resolvido!